docs/source/getting_started/getting_started.md

```diff
@@ -131,7 +148,7 @@ As a result, we will get something like:
 {
   "documents": [
   {
-      "id": "xayn_5283ef3",
+      "snippet_id": { "document_id": "xayn_5283ef3", "sub_id": 0 },
```

Comment:
Does this happen even if you never use the split option? I think that for backward compatibility if split is never used this example should stay the same.
If this is the case would be better to specify this and show it with two different examples.

Reply:
Ok, is specified underneath, but maybe we can find a way to specify both? 
In the ingestion example we never used "split = true".

Reply:
The `snippet_id` is  always returned the system doesn't know (or care) if `split` was used or not.

For backward compatibility we still return the previous `id` but it's deprecated in the OpenApi spec, so I didn't include it in the example.

Reply:
ok thanks.